src/Service.GraphQLBuilder/GraphQLTypes/SupportedTypes.cs

```diff
@@ -20,7 +20,10 @@ public static class SupportedTypes
         public const string DECIMAL_TYPE = "Decimal";
         public const string STRING_TYPE = "String";
         public const string BOOLEAN_TYPE = "Boolean";
+        public const string DATE_TYPE = "Date";
```

Comment:
The comment of this function says: "The type names mentioned here are HotChocolate Scalar built in types." I dont think DateTime2 is a supported type in HotChocolate, may need to modify the comment to reflect what this class means..